docs/reference/msbuild-targets.md

```diff
@@ -208,6 +208,8 @@ If using MSBuild to pack your project, use a command like the following:
 msbuild /t:pack <path to .csproj file> /p:NuspecFile=<path to nuspec file> /p:NuspecProperties=<> /p:NuspecBasePath=<Base path> 
 ```
 
+Please note that packing a nuspec using dotnet.exe or msbuild also leads to building the project by default. This can be avoided by passing ```--no-build``` property to dotnet.exe, which is the equivalent of setting ```<NoBuild>true</NoBuild> ``` in your project file.
```

Comment:
Have you tried this locally? It doesn't work. 

C:\Program Files\dotnet\sdk\2.2.0-preview1-007622\Sdks\Microsoft.NET.Sdk\build\Microsoft.PackageDependencyResolution.targets(327,5): error : Assets file 'C:\Users\jaredpar\temp\obj\project.assets.json' not found. Run a NuGet package restore to generate this file. [C:\Users\jaredpar\temp\example.csproj]
C:\Program Files\dotnet\sdk\2.2.0-preview1-007622\Sdks\Microsoft.NET.Sdk\build\Microsoft.PackageDependencyResolution.targets(167,5): error : Assets file 'C:\Users\jaredpar\temp\obj\project.assets.json' not found. Run a NuGet package restore to generate this file. [C:\Users\jaredpar\temp\example.csproj]

Reply:
updated, and provided an example too.